src/middleware/externalRequests.js

```diff
@@ -402,6 +404,10 @@ const addRequestQueryParameters = (ctx, request) => {
             path
           )
           break
+        default:
+          throw new Error(
```

Comment:
```suggestion
        default:
          ctx.statusCode = 500
          throw new Error(
```

Reply:
@bradsawadye  why not 400 - this appears to be a user input issue

```suggestion
        default:
          ctx.statusCode = 400
          throw new Error(
```

Reply:
This is more a configuration issue than a client sending a bad request. The client cant change anything in their payload to make this a valid request